PRINCIPLES.md

```diff
@@ -0,0 +1,254 @@
+<!-- see https://github.com/yzhang-gh/vscode-markdown/blob/master/README.md#table-of-contents -->
+# GitOps Principles v0.1.0 <!-- omit in toc -->
+
+## ⚠️ THIS DOCUMENT IS A WORK IN PROGRESS AND SUBJECT TO PUBLIC REVIEW BEFORE PUBLICATION <!-- omit in toc --> ⚠️ 
+
+This document provides a concrete definition of GitOps principles.
+
+The GitOps Principles are vendor and implementation neutral, and aim to provide a common understanding of GitOps systems, to enable a common framework of understanding beyond individual opinion. A second aim is to encourage innovation by clarifying the technical outcomes rather than the code, tests, or organizational elements needed to achieve them. 
```

Comment:
I struggle with `GitOps systems` as used in this statement - there are very meaningful components to GitOps that are not system rather process oriented. For instance if we consider the statement below: ```...a human or software agent will not interact with the running system and modify it directly. Instead, the agent will create a new declarative version of the desired state...``` states how the system must be interacted with.

> The GitOps Principles are vendor and implementation neutral, and aim to provide a common understanding of the specifications (qualities, characteristics, patterns?) that GitOps compliant systems and processes (workflows?) should satisfy.

Reply:
Agree with @murillodigital about `GitOps systems`. 
Maybe simplify this even more by making shorter sentences and be less vague:

```suggestion
The GitOps Principles are vendor and implementation neutral. 
They provide common patterns for GitOps compliant systems and processes.
```

Reply:
> They provide common patterns for GitOps compliant systems and processes. 

I am not sure that classifying some systems as GitOps compliant would be helpful in the long run. 

```suggestion
GitOps is a set of principles that governs how humans and technical systems interact in order to achieve the desired operational outcomes of repeatability, auditability, and visibility. 
```

Reply:
Have modified to integrate this discussion. Should hopefully resolve the lack of clarity.